reference/logical-replication.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Logical replication
+---
+
+# Logical replication
+
+PGroonga supports PostgreSQL built-in logical replication since 1.2.4. It requires PostgreSQL 10 or later.
+
+This document describes how to configure PostgreSQL built-in logical replication for PGroonga. Most of steps are normal steps. There are some PGroonga specific steps.
+
+It doesn't have to be have same schema for source of replication and
+destionation of replication for logical replication.
+
+Therefore, setting an index on only a destination of replication in here.
+
+## Summary
+
+Here are steps to configure PostgreSQL built-in logical replication for PGroonga. "[normal]" tag means that the step is a normal step for logical replication. "[special]" tag means that the step is a PGroonga specific step.
```

Comment:
Here are **the steps** to configure PostgreSQL built-in logical replication for PGroonga.?

_The_ is needed because **steps** is the concrete procedures.

---

ここにはstepsの前にtheがあってもなくても良いんですけど、PPGroongaのロジカルレプリケーションを作るのは具体的なこの手順で大丈夫ですよ、と読み手に伝えるにはtheがあった方がいいかなと思いました。